packages/core/test/hardhat/data-verification-mechanism/DesignatedVotingV2.js

```diff
@@ -151,12 +151,10 @@ describe("DesignatedVotingV2", function () {
 
     const price = 420;
     const salt = getRandomSignedInt();
-    // Note: the "voter" address for this vote must be the designated voting contract since its the one that will ultimately
-    // "reveal" the vote. Only the voter can call reveal through the designated voting contract.
     const hash = computeVoteHashAncillary({
       price,
       salt,
-      account: voter,
+      account: designatedVoting.options.address,
```

Comment:
note that this change here showcases some implications of this design change: the address in the hash, when using DesignatedVotingV2, is now the designated voting address as the designated voting contract is effectively the "staker".